### PR TITLE
minor code cleanup

### PR DESCRIPTION
**Pull request by aldenml**

### CODE REVIEW

File: src/storage.cpp [hunk 1]
```diff
@@ -301,7 +301,10 @@ namespace libtorrent {
 				m_part_file->export_file([&f, &ec](std::int64_t file_offset, span<char> buf)
 				{
 					iovec_t const v = {buf.data(), buf.size()};
-					std::int64_t const ret = f->writev(file_offset, v, ec.ec);
+#if TORRENT_USE_ASSERTS
```
Comment (arvidn) [inline src/storage.cpp]: if you instead to ``TORRENT_UNUSED(ret);`` you only add 1 line, instead of 4.

File: src/peer_connection.cpp [hunk 2]
```diff
@@ -5675,7 +5675,6 @@ namespace libtorrent {
 		}
 		if (size <= 0) return;
 
-		int i = 0;
```
Comment (arvidn) [inline src/peer_connection.cpp]: I thought I made this change already. is this from top of master?

File: src/storage.cpp (hunk 1 above)
Comment (aldenml) [inline src/storage.cpp]: yes, but for some reason I thought this was more "clear"

File: src/peer_connection.cpp (hunk 2 above)
Comment (aldenml) [inline src/peer_connection.cpp]: yes, on top of master

**Comment by aldenml** [comment]
I will do the changes, but I will wait for your `RC_1_1` merge is done, but you can put the changes in your PR if you want :)

**Comment by arvidn** [comment]
yeah, I exposed a bug in the storage refcounting in master, and fixing that uncovered another issue with (what appears to be) releasing the storage too early. This has made the merge take a bit longer, but I've already been bit again by the fact that you cannot rebase a merge commit, so I would like to hold off on new commits on top of master until I can sort this out and get my merge in

**Comment by arvidn** [comment]
I'm ready to merge this. did you intend to change that unused variable hunk?

**Comment by aldenml** [comment]
yes, in a few minutes I will push an update

File: include/libtorrent/buffer.hpp [hunk 3]
```diff
@@ -157,4 +157,4 @@ class buffer
 
 }
 
-#endif // BTORRENT_BUFFER_HPP_INCLUDED
```
Comment (arvidn) [inline include/libtorrent/buffer.hpp]: heh.. I wonder how this snuck in there..

**Pull request merged**